### ISSUE TITLE
PPV Streams

### ISSUE THREAD

**Issue by cyrillekey** [danamag/stremio-addons-list#735]
### Addon Manifest URL

https://stremio-addon-ppvstreams.vercel.app/manifest.json

### Addon Description

Watch live football, basketball,rugby and nfl matches and ppv events from ppv land .

### Language of Content

English (en-US)

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [X] http streams
- [X] live tv
- [ ] metadata
- [ ] misc
- [ ] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [ ] tv shows
- [ ] video games

**Comment by DrekAnon** [comment]
Ongoing UFC and NFL games don't seem to appear  in any of the categories despite them appearing on ppvland

**Comment by lebigboss59** [comment]
Put dislike for this racist

**Comment by Denmark4030** [comment]
> Put dislike for this racist
I’m trying to report it to federal bureau of investigation

**Comment by Denmark4030** [comment]
> Put dislike for this racist
> I’m trying to report it to federal bureau of investigation

Informed Stremio their  reply was it’s someone from community addons told me go to Reddit

**Comment by DrekAnon** [comment]
> > Put dislike for this racist
> > I’m trying to report it to federal bureau of investigation

you reported that you were trying to watch ppv streams for free and someone changed it to a racist GIF to the FBI?

**Comment by Denmark4030** [comment]
Yes that’s 100% correct it is racist, and whoever says anything different is wrong 
And anyway the addon is free , but I uninstalled it, yes it’s raciest, and not good enough

**Comment by Denmark4030** [comment]
YES ITS RACIEST PLEASE PUT DISLIKE

**Comment by cdkao** [comment]
I got the little pink pig

**Comment by BRKsays** [comment]
For posterity, @cyrillekey is not to be blamed according to a reddit post discussion. The regards at ppvland are the racist one and all these is because of them.
